src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -97,6 +97,9 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <p>
```

Comment:
Combine with next paragraph

Reply:
It would be better to leave as it is I guess, as next paragraph gives an example, not config.

Reply:
https://checkstyle.org/config_whitespace.html#EmptyLineSeparator

we already have `To configure the check`
but for some reason example goes above config.
we need to swap them

![image](https://user-images.githubusercontent.com/812984/90020280-7f5f1700-dc64-11ea-9717-1677a5696c5e.png)

Reply:
Done